src/JavascriptVM/JavascriptVM.tsx

```diff
@@ -103,8 +105,8 @@ export const VMProvider: FunctionComponent = ({ children }) => {
 
           syncExecutionState();
 
-          setInterpreter(
-            new BlocklyInterpreter(code, {
+          try {
```

Comment:
Maybe separate the callback definitions out of the try block? Makes it a little more clear it’s the code we think might fail (creating the BlocklyInterpreter)